src/components/calcite-slider/calcite-slider.scss

```diff
@@ -114,248 +145,252 @@ $tick-height: 4px;
+:host([scale="l"]) {
+  .thumb--value {
+    transform: translate(8px, var(--calcite-slider-thumb-offset));
   }
   .thumb--minValue {
-    transform: translate(-50%, -2px);
+    transform: translate(-8px, var(--calcite-slider-thumb-offset));
   }
 }
-:host([ticks][precise]:not([precise="false"])) {
+
+:host([label-handles]) {
   .thumb {
-    transform: translate(7px, -20px);
+    transform: translate(50%, -30px);
   }
   .thumb--minValue {
-    transform: translate(-7px, -3px);
+    transform: translate(-50%, -30px);
   }
-}
-:host([has-histogram][ticks][precise]:not([precise="false"])) {
-  .thumb {
-    transform: translate(7px, -3px);
+  &:host([scale="s"]:not([has-histogram])) {
+    .thumb {
+      transform: translate(50%, -23px);
+    }
+    .thumb--minValue {
+      transform: translate(-50%, -23px);
+    }
   }
-  .thumb--minValue {
-    transform: translate(-50%, -3px);
+  &:host([scale="l"]:not([has-histogram])) {
+    .thumb {
+      transform: translate(50%, -32px);
+    }
+    .thumb--minValue {
+      transform: translate(-50%, -32px);
+    }
   }
 }
-:host([label-handles][precise]:not([precise="false"])) {
+
+:host([has-histogram][label-handles]) {
   .thumb {
-    transform: translate(50%, -38px);
-  }
-  .thumb--minValue {
-    transform: translate(-50%, -2px);
+    transform: translate(50%, var(--calcite-slider-thumb-offset));
   }
-}
-:host([has-histogram][label-handles][precise]:not([precise="false"])) {
-  .thumb {
-    transform: translate(50%, -2px);
+  .handle__label {
+    margin-top: 0.5em;
   }
   .thumb--minValue {
-    transform: translate(-50%, -2px);
+    transform: translate(-50%, var(--calcite-slider-thumb-offset));
   }
 }
-:host([ticks][label-handles][precise]:not([precise="false"])) {
-  .thumb {
-    transform: translate(50%, -37px);
+
+:host([label-handles][precise]) {
+  &:host(:not([has-histogram])[scale="s"]) .thumb--value {
+    transform: translate(50%, -34px);
   }
-  .thumb--minValue {
-    transform: translate(-50%, -3px);
+  &:host(:not([has-histogram])[scale="m"]) .thumb--value {
+    transform: translate(50%, -44px);
   }
-}
-:host([has-histogram][ticks][label-handles][precise]:not([precise="false"])) {
-  .thumb {
-    transform: translate(50%, -3px);
+  &:host(:not([has-histogram])[scale="l"]) .thumb--value {
+    transform: translate(50%, -50px);
   }
-  .thumb--minValue {
-    transform: translate(-50%, -3px);
+  &:host(:not([has-histogram])) .thumb--minValue {
+    transform: translate(-50%, var(--calcite-slider-thumb-offset));
+    .handle__label {
+      margin-top: 0.5em;
+    }
+  }
+  &:host(:not([has-histogram])) .thumb--value {
+    .handle__label {
+      margin-bottom: 0.5em;
+    }
   }
 }
 
 .thumb:focus,
 .thumb--active {
-  z-index: 3;
+  // z-index: 3;
   .handle {
-    background-color: var(--calcite-ui-brand);
-    @include shadow(1, "press");
+    @apply bg-brand;
+    box-shadow: 0 0 8px 0 rgb(0 0 0 / 16%); // TO-DO: should this be shadow-1-press in TW config?
```

Comment:
Commented on the box-shadows issue here: https://github.com/Esri/calcite-components/issues/1431#issuecomment-926155647 (now I'll remove this note.)